packages/devtools_app/lib/src/shared/history_viewport.dart

```diff
@@ -78,8 +78,8 @@ class _HistoryViewportState<T> extends State<HistoryViewport<T?>> {
   Widget _buildTitle(BuildContext context, ThemeData theme) {
     final title = widget.generateTitle == null
         ? '  '
-        : widget.generateTitle!(widget.history.current.value);
-    final defaultTitleStyle = theme.textTheme.subtitle2;
+        : widget.generateTitle!(widget.history.current.value!);
```

Comment:
same comment here. widget.history.current.value could be null, so let's remove the !

Reply:
done